src/app/shared/models/statement-anchors.model.ts

```diff
@@ -1,12 +1,13 @@
 export enum StatementAnchors {
-  MOST_IMPORTANT = 'najwazniejsze-informacje',
-  PHONES = 'telefony',
   ACCOMMODATION = 'noclegi-i-mieszkania',
-  MATERIAL_AID = 'pomoc-materialna',
-  TRANSPORT = 'transport',
+  EDUCATION = 'edukacja',
   HEALTH = 'zdrowie',
+  JOB = 'praca',
   LEGAL_HELP = 'pomoc-prawna',
+  MATERIAL_AID = 'pomoc-materialna',
+  MOST_IMPORTANT = 'najwazniejsze-informacje',
+  PHONES = 'telefony',
   TRANSLATIONS = 'tlumaczenia',
-  JOB = 'praca',
-  EDUCATION = 'edukacja',
+  TRANSPORT = 'transport',
+  WORK = 'praca',
```

Comment:
`WORK = 'praca'` do usunięcia, jest już `JOB = 'praca'`

Reply:
Usunięto.